.github/workflows/tests.yml

```
@@ -13,14 +13,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9, '3.10']
+        python-version: [3.8, 3.9, 3.10, 3.11, 3.12, '3.13-dev']
```

```suggestion
        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13']
```

We need the quotes, otherwise 3.10 is treated as a number and becomes 3.1. And then it's just neater to quote all of them.